docs/code/MlNetCookBook.md

```diff
@@ -595,7 +595,7 @@ As a general rule, *if you use a parametric learner, you need to make sure your
 
 ML.NET offers several built-in scaling algorithms, or 'normalizers':
 - MinMax normalizer: for each feature, we learn the minimum and maximum value of it, and then linearly rescale it so that the values fit between -1 and 1.
-- MeanVar normalizer: for each feature, compute the mean and variance, and then linearly rescale it to zero-mean, unit-variance.
+- MeanVariance normalizer: for each feature, compute the mean and variance, and then linearly rescale it to zero-mean, unit-variance.
```

Comment:
>MeanVariance [](start = 2, length = 12)

In the field, we usually use the term `Standardize` to reflect this normalization technique. This is very "statistics-y", but it does seem to be standard. How would everyone feel about changing "MeanVariance" to "Standardize", or at least offer a "Standardize" alias?

Reply:
I don't feel MVN is super bad because it's already an operator in neural networks (e.g., ONNX and Caffe).

---
In reply to: [264826182](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264826182) [](ancestors = 264826182)

Reply:
That's a good point. Plus it's more precise than `Standardize`. Let's keep it MVN. We can always add an alias for `Standardize` if people are up in arms.

---
In reply to: [264891367](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264891367) [](ancestors = 264891367,264826182)